### PR TITLE
xcpretty support

### PR DESCRIPTION
**Pull request by madhurig**

### CODE REVIEW

File: Tasks/Common/ios-signing-common/ios-signing-common.ts [hunk 1]
```diff
@@ -18,31 +18,26 @@ export async function installCertInTemporaryKeychain(keychainPath : string, keyc
     await deleteKeychain(keychainPath);
 
     //create keychain
-    var createKeychainCommand : ToolRunner =  tl.createToolRunner(tl.which('security', true));
-    createKeychainCommand.arg(['create-keychain', '-p', keychainPwd]);
-    createKeychainCommand.pathArg(keychainPath);
+    var createKeychainCommand : ToolRunner =  tl.tool(tl.which('security', true));
```
Comment (madhurig) [inline Tasks/Common/ios-signing-common/ios-signing-common.ts]: Changes are needed as Xcode task moved to newer version of vsts-task-lib

File: Tasks/Xcode/package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
   "devDependencies": {
-    "vsts-task-lib": "0.8.5"
+    "vsts-task-lib": "0.9.14"
```
Comment (madhurig) [inline Tasks/Xcode/package.json]: This has to be updated to new published version of vsts-task-lib with the pipe command changes

File: Tasks/Xcode/task.json [hunk 3]
```diff
@@ -215,6 +215,15 @@
             "groupName": "advanced"
         },
         {
+            "name": "useXcpretty",
+            "type": "boolean",
+            "label": "Use xcpretty",
+            "required": false,
+            "defaultValue": false,
+            "helpMarkDown": "Use xcpretty to format xcodebuild output and generate junit test results report. Requires xcpretty be installed on agent hosts. See [xcpretty](https://github.com/supermarin/xcpretty) for details.",
```
Comment (DavidStaheli) [inline Tasks/Xcode/task.json]: "junit" should be "JUnit"

**Comment by bryanmacfarlane** [comment]
remember we already branched for releases 106.  So you will need to cherry pick.

**Pull request merged**